### DISCUSSION TITLE
Add language selector to WYSIWYG

### DISCUSSION THREAD

**Discussion by narduin** [directus/directus#13208] [Archive]
Hello!

I'm trying to add the toolbar option `content_langs` to my WYSIWYG fields via Directus' `options override`.

I've read [the documentation](https://www.tiny.cloud/docs/tinymce/6/content-localization/#content_langs) but can't figure out a way to implement this correctly.

I've successfully replaced everything in the toolbar with the language button but **I'd like to add it rather than replace** the other buttons with this:

```json
{
    "toolbar": "language",
    "content_langs": [
        {
            "title": "English",
            "code": "en"
        },
        {
            "title": "Spanish",
            "code": "es"
        },
    ]
}
```

Do I have to manually recreate the toolbar in the `options override` or is there a way to add a button to an existing toolbar?
Thanks!

**Comment by rijkvanzanten** [comment]
Right now, the configured overrides are assigned on top of the rest of the "regular" setup, so configuring `toolbar` from the overrides will override Directus' default (configured) toolbar:

https://github.com/directus/directus/blob/51e90f6c4cbd1c3e9cd9e2b000eef694812280d5/app/src/interfaces/input-rich-text-html/input-rich-text-html.vue#L345-L348

You could hack in the custom button by editing the raw value that's saved for the toolbar option as follows:

https://user-images.githubusercontent.com/9141017/167658772-03ed29b4-47af-46a2-80e9-29bf2b1a6cb4.mp4

**Comment by narduin** [comment]
This with the `content_langs` option works perfectly!
Thanks :)

Full `options override` code:

```json
{
    "content_langs": [
        {
            "title": "English",
            "code": "en"
        },
        {
            "title": "Spanish",
            "code": "es"
        }
    ]
}
```
